### PR TITLE
[Obvious Fix] Fix typos, formatting and escaping in reference documentation

### PR DESCRIPTION
**Pull request by bianjp**
Fix various typos, formatting and escaping issues that I find while reading through the reference documentation.

This is an obvious fix.

Note: Some symbol combinations (including "<=") serve as textual symbol replacements in AsciiDoc, and require escaping when replacement isn't expected. See http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#text-replacement

### CODE REVIEW

File: src/docs/asciidoc/core/core-aop.adoc [hunk 1]
```diff
@@ -545,14 +545,14 @@ this purpose. A typical such aspect would look as follows:
 		 *
 		 * If you group service interfaces by functional area (for example,
 		 * in packages com.xyz.someapp.abc.service and com.xyz.someapp.def.service) then
-		 * the pointcut expression "execution(* com.xyz.someapp..service.*.*(..))"
+		 * the pointcut expression "execution(* com.xyz.someapp.service.*.*(..))"
```
Comment (dreis2211) [inline src/docs/asciidoc/core/core-aop.adoc]: Without looking too deep into it, I believe the "fix" here is wrong as using two-dots is actually intended in order to match both the `abc` and `def` package.

File: src/docs/asciidoc/core/core-aop.adoc [hunk 2]
```diff
@@ -545,14 +545,14 @@ this purpose. A typical such aspect would look as follows:
...
 		 * could be used instead.
 		 *
 		 * Alternatively, you can write the expression using the 'bean'
 		 * PCD, like so "bean(*Service)". (This assumes that you have
 		 * named your Spring service beans in a consistent fashion.)
 		 */
-		@Pointcut("execution(* com.xyz.someapp..service.*.*(..))")
+		@Pointcut("execution(* com.xyz.someapp.service.*.*(..))")
```
Comment (dreis2211) [inline src/docs/asciidoc/core/core-aop.adoc]: Same as above, I believe

File: src/docs/asciidoc/core/core-aop.adoc (hunk 1 above)
Comment (bianjp) [reply src/docs/asciidoc/core/core-aop.adoc]: You're right. I just amended the commit.

**Comment by rstoyanchev** [comment]
Thanks for the updates! The changes are now in master.

**Pull request closed without merging**